### ISSUE TITLE
[Bug] "vstshortcuts" disable switch not respected in Serum

### ISSUE THREAD

**Issue by kwvg** [LiveEnhancementSuite/LESforMacOS#7]
**Describe the bug**

Split from #5. Reported by @ptrkstr 

Original Description:

> vstshortcuts = 0 but when I open serum, pressing 1, 2, 3, 4 switches the tabs up the top.

**To Reproduce**

>    1. Open LES
>    2. Open LES settings and set vstshortcuts = 0
>    3. Open Live 11
>    4. Insert serum.vst
>    5. Press 1,2,3,4

**Expected behavior**
>  Tabs don't switch.

**Desktop (please complete the following information):**

>   * macOS Big Sur 11.2.3 (20D91)
>   * LES v1.3.1-alpha (0.9.78 in mac inspector)
>   * Live 11.0 (2021-02-08_9dc150af94)

**Additional context**

>  I've restarted my mac and problem still persists.

**Comment by kwvg** [comment]
@ptrkstr 

Have you experienced that with Serum alone or is this a plugin agnostic bug? I cannot find any documentation specifying keys 1 - 4 being official keyboard shortcuts in Serum but apparently they don't document shortcuts much and the official references to them aren't the manual but the forums and even that seems incomplete according to Reddit.

**Comment by ptrkstr** [comment]
Hey @kittywhiskers, thanks for getting in touch. They're not official shortcuts of serum, but shortcuts created by LES. Here's a link to the documentation:

https://docs.enhancementsuite.me/keyboard_shortcuts.html#vst-shortcuts

So it says the shortcuts only activate when the plugin is focused but what ends up happening is outside of live, those buttons no longer work (ie typing 1,2 etc in a TextEdit box).

So then a workaround may be to disable the shortcuts using the flag in settings but that seems to also not work, which leaves the only solution being to quit LES.

**EDIT** Oh dear, I misunderstood what you wrote. To answer; I'm not sure if it's an official shortcut now in serum. But I don't think so given that the shortcuts stop working once LES is quit? But it's been a while so I'll verify if the shortcuts stop working once LES is quit.

**Comment by Inversil** [comment]
yeah these are LES shortcuts; it's been broken in the mac version for a while but it's part of [HSModule. ](https://github.com/LiveEnhancementSuite/HSModule)
This was usually my job to work on, but apple has locked me out of running newer versions of the program, so I can no longer actively develop the script.
